pillarbox-demo/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,6 @@
     <string name="app_name">PillarboxDemo</string>
     <string name="hello_world">Hello Pillarbox!</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SRGSSR/pillarbox-android/blob/9e672e88664d24e42b2fbb65a69ff4507dff46b6/pillarbox-demo/src/main/res/values/strings.xml#L7"></span>The resource <code>R.string.hello_world</code> appears to be unused</td>
    </tr>
  </tbody>
</table>